spark-common/src/main/js/src_contentwin/spark-contentwindow.js

```diff
@@ -29,9 +33,27 @@ window.SPARK = window.SPARK || {};
 
                     document.head.appendChild(iframeThemeStyle);
                 })
+            } else {
+                var contextPathTag = parentWindow.document.querySelector('meta#confluence-context-path');
```

Comment:
We should use the `ajs-context-path` meta tag instead, which also works on Jira:

Confluence:
```
<meta name="ajs-context-path" content="/confluence">
```

Jira (cloud):
```
<meta name='ajs-context-path' content=''>
```